test/test_querying.py

```diff
@@ -83,6 +83,23 @@ def test_get_experiments(session):
     )
     assert_frame_equal(r, df)
 
+    metadata_keys = ['root_dir', 'contact', 'email', 'created', 'description', 'notes']
+
+    # Won't try and match everything, there is not much useful metadata, just 
+    # check dimensions are correct. Metadata correctness checked in test_metadata
+    for k in metadata_keys:
+        r = cc.querying.get_experiments(session, k=True)
```

Comment:
Does this actually work? This would have to come as `**{k: True}`, wouldn't it?

Reply:
Yes. Thanks. It "works" in a trivial way, that does not actually select for the column :)